classify/pose_classify.py

```diff
@@ -210,7 +212,6 @@ def landmarks_to_embedding(landmarks_and_scores):
     embedding = keras.layers.Flatten()(landmarks)
     return embedding
 
-
 # Define the model
```

Comment:
I see a big chunk of global code and wondering if this could be tucked into some function say model_init or even main?